sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -90,10 +90,14 @@ def begin_recognize_receipts(self, receipt, **kwargs):
         :keyword int polling_interval: Waiting time between two polls for LRO operations
             if no Retry-After header is present. Defaults to 5 seconds.
         :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword str locale: Locale of the receipt. Supported locales include: en-AU, en-CA, en-GB, en-IN,
```

Comment:
nit: can we include the default at the beginning, i.e. Default is en-US, other supported locales include..., this way the default is more easy to read?

Reply:
Sure